client/me/account/controller.js

```diff
@@ -11,32 +11,24 @@ import i18n from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import analytics from 'lib/analytics';
 import userSettings from 'lib/user-settings';
 import { setDocumentHeadTitle as setTitle } from 'state/document-head/actions';
 import AccountComponent from 'me/account/main';
 import username from 'lib/username';
 
-const ANALYTICS_PAGE_TITLE = 'Me';
-
 export default {
 	account( context, next ) {
-		const basePath = context.path;
 		let showNoticeInitially = false;
 
-		context.store.dispatch( setTitle( i18n.translate( 'Account Settings', { textOnly: true } ) ) ); // FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+		// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
+		context.store.dispatch( setTitle( i18n.translate( 'Account Settings', { textOnly: true } ) ) );
```

Comment:
It's a bit odd here to touch these lines if only to move the comment :) Let's either update it as suggested or leave it be. Were we hitting a max-length linter issue?

Reply:
> Were we hitting a max-length linter issue?

Yup.
I think I got a commit rejected in one of the previous analytics PRs because of this, and then I just went auto-pilot.
Updating as suggested is quite easy to be fair, but it also adds so much noise to these already messy PRs.

Reply:
I will give you the 👍 to ignore old linter errors in this case :) You can still commit with `git commit --no-verify` (please use responsibly) and we can verify in ci manually to see that units have passed and no new lint errors have creeped in.